plugins/available/explain.plugin.bash

```diff
@@ -0,0 +1,23 @@
+cite about-plugin
+about-plugin 'mankier.com explain function to explain other commands'
+
+# Add this to ~/.bash_profile or ~/.bashrc
+explain () {
+  if [ "$#" -eq 0 ]; then
```

Comment:
Can you add the function header/meta function calls (`about`, `group`, `param`, `example`) here?